buildpacks/python.go

```diff
@@ -33,119 +32,120 @@ func (bt PythonBuildTool) Version() string {
 	return bt.version
 }
 
-func (bt PythonBuildTool) pyenvInstallDir() string {
-	return filepath.Join(bt.spec.SharedCacheDir, "pyenv")
+func (bt PythonBuildTool) AnacondaInstallDir() string {
+	return filepath.Join(bt.spec.SharedCacheDir, "miniconda3", fmt.Sprintf("miniconda-%s", AnacondaToolVersion))
 }
 
-func (bt PythonBuildTool) pythonInstallDir() string {
-	return filepath.Join(bt.pyenvInstallDir(), "versions", bt.Version())
-}
-func (bt PythonBuildTool) pkgVirtualEnvDir() string {
-	return filepath.Join(bt.spec.PackageCacheDir, "python", bt.Version())
+func (bt PythonBuildTool) EnvironmentDir() string {
+	return filepath.Join(bt.spec.PackageCacheDir, "conda-python", bt.Version())
 }
 
-/*
-TODO: Install libssl-dev (or equivalent / warn) and zlib-dev based on platform
-*/
 func (bt PythonBuildTool) Install() error {
-	pyenvGitUrl := "https://github.com/pyenv/pyenv.git"
-	pyenvDir := bt.pyenvInstallDir()
-	pythonVersionDir := bt.pythonInstallDir()
-	virtualenvDir := bt.pkgVirtualEnvDir()
+	anacondaDir := bt.AnacondaInstallDir()
+	setupDir := bt.spec.PackageDir
 
-	bt.InstallPlatformDependencies()
-
-	if _, err := os.Stat(pyenvDir); err == nil {
-		fmt.Printf("pyenv installed in %s\n", pyenvDir)
+	if _, err := os.Stat(anacondaDir); err == nil {
+		fmt.Printf("anaconda installed in %s\n", anacondaDir)
 	} else {
-		fmt.Printf("Installing pyenv\n")
+		fmt.Printf("Installing anaconda\n")
 
-		_, err := git.PlainClone(pyenvDir, false, &git.CloneOptions{
-			URL:      pyenvGitUrl,
-			Progress: os.Stdout,
-		})
+		downloadUrl := bt.DownloadUrl()
 
+		fmt.Printf("Downloading Miniconda from URL %s...\n", downloadUrl)
+		localFile, err := DownloadFileWithCache(downloadUrl)
 		if err != nil {
-			fmt.Printf("Unable to clone pyenv!\n")
-			return fmt.Errorf("Couldn't clone pyenv: %v\n", err)
+			fmt.Printf("Unable to download: %v\n", err)
+			return err
 		}
-	}
 
-	if _, err := os.Stat(pythonVersionDir); err == nil {
-		fmt.Printf("Python %s installed in %s\n", bt.Version(), pythonVersionDir)
-	} else {
-		os.Setenv("PYENV_ROOT", pyenvDir)
-		PrependToPath(filepath.Join(pyenvDir, "bin"))
+		// TODO: Windows
```

Comment:
I think we could trigger `Linux for Windows Subsystem` to test this, we can open another issue for that.

Reply:
Not everyone uses WSL, we should make sure we have an install / setup that works on pure Windows :)